tests/phpunit/tests/test-multisite-option.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+if ( is_multisite() ) :
+
+	class Multisite_Option_Test extends PLL_UnitTestCase {
+
+		public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+			parent::wpSetUpBeforeClass( $factory );
+
+			self::create_language( 'en_US' );
+
+			require_once POLYLANG_DIR . '/include/api.php';
+		}
+
+		public function set_up() {
+			parent::set_up();
+
+			$links_model = self::$model->get_links_model();
+			$GLOBALS['polylang'] = new PLL_Admin( $links_model );
+		}
+
+		public function tear_down() {
+			parent::tear_down();
+
+			unset( $GLOBALS['polylang'] );
+		}
+
+		/**
+		 * @ticket #1685
+		 * @see https://github.com/polylang/polylang-pro/issues/1685
+		 */
+		public function test_translate_blogname() {
+			$blog_id = self::factory()->blog->create( array( 'title' => 'The new blog to test' ) );
+
+			$language = self::$model->get_language( 'en' );
```

Comment:
The intermediate variable `$language` doesn’t seem necessary.

Reply:
Yes. I also removed other useless code in https://github.com/polylang/polylang/pull/1284/commits/6e21e008d4b295fd0e5ef4eebb1efb67b746532b